apps/geomat/src/app/components/cdk-drag-drop-sorting-example/cdk-drag-drop-sorting-example.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Why did you add this empty component. It is not needed. Please remove it.

Reply:
I removed it.